pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -55,6 +55,15 @@ func (tc *TidbCluster) PDImage() string {
 	return image
 }
 
+func (tc *TidbCluster) PDVersion() string {
+	version := tc.Spec.PD.Version
+	if version == nil {
+		version = &tc.Spec.Version
+	}
+
+	return *version
+}
```

Comment:
I suggest letting pd tell operator the PDVersion (Don't know how or whether it exists) instead of judging it from `spec.pd.version` as we can't solve the case like version: nightly.

Reply:
I don't how to let pd tell its version before starting it.